spec/support/capybara_screenshot.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "capybara-screenshot/rspec"
```

Comment:
This require looks odd

Reply:
> This require looks odd

It's referring to something that's a part of the capybara-screenshot lib?

Reply:
They tell us to do this on their readme so 🤷‍♀️

Reply:
it is part of `capybara-screenshot` yes 🤔